Updating bonsai historical block limit to be more clear (#1548)
hyperledger/besu-docs: docs/public-networks/reference/cli/options.md

</Tabs>

When using [Bonsai Tries](../../concepts/data-storage-formats.md#bonsai-tries), the [maximum number of previous blocks](../../concepts/data-storage-formats.md#accessing-data) for which Bonsai can reconstruct a historical state. The default is 512. 

:::note

> If you plan on querying historical blocks or state via the [RPC-API](../api/index.md), you may need to adjust the default value or your configured value to avoid errors. 

Reviewer: ```suggestion
If you plan on querying historical blocks or state using the [JSON-RPC API](../api/index.md), you might need to adjust the default value or your configured value to avoid errors. 
```
